packages/yourloops/pages/patient/teams/page.tsx

```diff
@@ -157,7 +158,11 @@ function PatientTeamsPage(props: PatientTeamsPageProps): JSX.Element | null {
     if (loading) {
       setLoading(false);
     }
-  }, [teamHook.initialized, teamHook.errorMessage, loading, errorMessage]);
+  }, [teamHook.initialized, teamHook.errorMessage, loading, errorMessage, t]);
+
+  React.useEffect(() => {
+    setPageTitle(t("menu-medical-teams"));
```

Comment:
`your-loops.com/patient/xxx/teams` has to `My care teams | YouLoops`
```suggestion
    setPageTitle(t("breadcrumb-patient-teams"));
```